package.json

```diff
@@ -54,194 +54,157 @@
 	},
 	"dependencies": {
 		"@apollo/react-hooks": "^3.1.3",
-		"@babel/core": "^7.8.6",
-		"@blueprintjs/core": "^3.24.0",
-		"@blueprintjs/datetime": "^3.15.2",
-		"@blueprintjs/select": "^3.12.0",
 		"@eventespresso/react-exit-modal-typeform": "^1.0.4",
-		"@svgr/webpack": "5.2.0",
-		"@wordpress/components": "^9.2.3",
+		"@wordpress/components": "^9.2.6",
+		"@wordpress/compose": "^3.11.0",
 		"@wordpress/i18n": "^3.9.0",
-		"@wordpress/is-shallow-equal": "^1.8.0",
-		"accounting-js": "^1.1.1",
-		"antd": "^4.0.1",
+		"@wordpress/keycodes": "^2.9.0",
+		"@wordpress/url": "^2.11.0",
+		"@wordpress/viewport": "^2.13.2",
+		"antd": "^4.1.0",
 		"apollo-cache-inmemory": "^1.6.5",
 		"apollo-client": "^2.6.8",
 		"apollo-link-http": "^1.5.16",
-		"babel-jest": "^25.1.0",
-		"babel-loader": "8.0.6",
-		"camelcase": "^5.3.1",
-		"case-sensitive-paths-webpack-plugin": "2.3.0",
 		"classnames": "^2.2.6",
 		"cuid": "^2.1.8",
-		"date-fns": "^2.10.0",
+		"date-fns": "^2.11.1",
 		"date-fns-tz": "^1.0.10",
-		"decimal.js-light": "^2.5.0",
-		"dotenv": "8.2.0",
-		"dotenv-expand": "5.1.0",
-		"file-loader": "5.1.0",
 		"final-form": "^4.19.0",
 		"final-form-arrays": "^3.0.2",
-		"final-form-calculate": "^1.3.1",
-		"fs-extra": "^8.1.0",
 		"graphql-tag": "^2.10.3",
-		"html-webpack-plugin": "4.0.0-beta.5",
-		"immutable": "^4.0.0-rc.12",
 		"invariant": "^2.2.4",
-		"js-green-licenses": "^1.1.0",
-		"memize": "^1.0.5",
-		"moment": "^2.24.0",
-		"moment-duration-format": "^2.3.2",
-		"moment-timezone": "^0.5.28",
-		"node-sass": "^4.13.1",
-		"npm-watch": "^0.6.0",
-		"optimize-css-assets-webpack-plugin": "5.0.3",
-		"pluralize": "^8.0.0",
-		"pnp-webpack-plugin": "1.6.4",
-		"postcss-flexbugs-fixes": "4.2.0",
-		"postcss-normalize": "8.0.1",
-		"postcss-preset-env": "6.7.0",
-		"postcss-safe-parser": "4.0.2",
-		"prop-types": "^15.7.2",
-		"querystringify": "^2.1.1",
 		"ramda": "^0.27.0",
 		"react": "^16.13.1",
 		"react-app-polyfill": "^1.0.6",
 		"react-beautiful-dnd": "^13.0.0",
-		"react-currency-formatter": "^1.1.0",
-		"react-dev-utils": "^10.2.0",
 		"react-dom": "^16.13.1",
 		"react-final-form": "^6.4.0",
 		"react-final-form-arrays": "^3.1.1",
-		"react-final-form-listeners": "^1.0.2",
-		"react-select": "^3.0.8",
 		"react-spring": "^8.0.27",
-		"rememo": "^3.0.0",
```

Comment:
there was not only this, but also `memize` :smile:

Reply:
Got rid of both :)